src/server/db/db_helper.py

```diff
@@ -488,7 +496,8 @@ async def create_schema_in_db(self, db_name, schema):
 
     async def get_tables_from_schema_id(self, schema_id):
         try:
-            con = sqlite3.connect(f'{db_name}.db')
+            db_file_path = os.path.join(self.temp_db_folder, f'{db_name}.db')
```

Comment:
you are getting the db name as schema_id but you are using db_name as the file name. Change it schema_id

Reply:
done